qa/automation/ScoringPortal.Tests.Scenarios/1. Access to Portal/1.1. Access to Career Portal.feature

```diff
@@ -1,6 +1,67 @@
 Feature: 1.1. Access to Career Portal
 
-Scenario: 1.1_01 User can open Career Portal
-	Given 
-	When 
-	Then 
+Background:

```

Comment:
Why do you use the old file? 
Our HW includes creating a new feature file with your scenarios, why did you decide to reuse an old file with the incorrect name?